cohorts/load.py

```diff
@@ -307,7 +304,28 @@ def plot_init(self, on, col, col_equals):
             else:
                 return col_func(self, on, col, col_equals)
 
-    def plot_benefit(self, on, col=None, col_equals=None):   
+    def plot_benefit(self, on, col=None, col_equals=None):
+        """Plot a comparison of benefit/response in the cohort on a given variable
+
+        If the variable (through `on` or `col` is binary) this will compare
+        odds-ratios and perform a Fisher's exact test.
+
+        If the variable is numeric, this will compare the distributions through
+        a Mann-Whitney test and plot the distributions with box-strip plot
+
+        Parameters
+        ----------
+        on : See `cohort.load.plot_init`
+        col : str, optional
+            If `on` is not specified, the column name on which to split the cohort
```

Comment:
Yeah, basically; or `on` can just be a string name for a new column generated by comparing `col` with `col_equals`